File: streaming-server/src/main/java/io/github/hs96wings/streaming_server/member/repository/MemberRepository.java
package io.github.hs96wings.streaming_server.member.repository;

import io.github.hs96wings.streaming_server.member.domain.Member;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.Optional;

public interface MemberRepository extends JpaRepository<Member, Long> {
    Optional<Member> findByUserid(String userid);
}

File: streaming-server/src/main/java/io/github/hs96wings/streaming_server/member/repository/MemberRepository.kt
package io.github.hs96wings.streaming_server.member.repository

import io.github.hs96wings.streaming_server.member.domain.Member
import org.springframework.data.jpa.repository.JpaRepository
import java.util.*

interface MemberRepository : JpaRepository<Member, Long> {
    fun findByUserid(userid: String?): Optional<Member>
}
